src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/MetagenomicsProgramGroup.java

```diff
@@ -3,9 +3,14 @@
 import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
 import org.broadinstitute.hellbender.utils.help.HelpConstants;
 
-public final class FastaProgramGroup implements CommandLineProgramGroup {
+/**
+ * Tools that detect pathogens, e.g. bacteria.
```

Comment:
Description should be more general

-> Tools that perform metagenomic analysis, e.g. microbial community composition and pathogen detection

Reply:
Done.

Reply:
Done.